tests/testthat/test-utils-plumber.R

```diff
@@ -310,7 +310,7 @@ test_that("citation_from_version works as expected", {
 
   expect_equal(names(citation), c("citation", "version_id", "date_accessed"))
   expect_equal(citation$citation,
-               "World Bank (2022), Poverty and Inequality Platform (version 20220609_2011_02_02_PROD) [data set]. pip.worldbank.org. Accessed on 2022-12-05")
+               paste0("World Bank (2022), Poverty and Inequality Platform (version 20220609_2011_02_02_PROD) [data set]. pip.worldbank.org. Accessed on ", as.character(Sys.Date())))

```

Comment:
Why is the version ID hardcoded? I know this is the test, but will we have to change it each time we release new data?

Reply:
No, the actual citation is created based on this `version_id`, so no need to update it with each release. This test should need no update, unless we change the structure of the `version_id`